test/integration/src/test/java/io/pravega/test/integration/KeyValueTableTest.java

```diff
@@ -192,6 +218,276 @@ public void testCreateDeleteKeyValueTable() {
+        log.info("Version list1: {}", versionList1);
+        assertNotNull("Version list1 should not be empty", versionList1);
+        assertEquals("Version list1 size should be same as entry list1 size",
+                versionList1.size(), entryMap1.size());
+
+        List<Integer> keyList = new ArrayList<>();
+        for (int i = 0; i < keyArray.length; i++) {
+            keyList.add(keyArray[i]);
+        }
+        log.info("Key list: {}", keyList);
+        List<TableEntry<Integer, String>> getEntryList1 = newKeyValueTable.getAll(keyFamily, keyList).join();
+        log.info("Get Entry List1: {}", getEntryList1);
+        assertNotNull("Get Entry List1 should not be empty", getEntryList1);
+        assertEquals("Get Entry List1 size should be same as get entry list1 size",
+                getEntryList1.size(), entryMap1.size());
+        List<Integer> entryKeyList = new ArrayList<>();
+        for (Integer key: entryMap1.keySet()) {
+            entryKeyList.add(key);
+        }
+        log.info("entryKeyList: {}", entryKeyList);
+        for (int i = 0; i < keyArray.length; i++) {
+            assertEquals("Corresponding key for getEntryList should be as inserted",
+                    keyList.get(i), getEntryList1.get(i).getKey().getKey());
+            assertEquals("Corresponding key for entryList should be as inserted",
+                    entryKeyList.get(i), getEntryList1.get(i).getKey().getKey());
+        }
+
+        // Comparing each & every entry values of both entryList and getEntryList.
+        for (int i = 0; i < keyArray.length; i++) {
+            assertEquals("All string values of both entryList1 and getEntryList1 should be equal",
+                    entryMap1.get(entryKeyList.get(i)), getEntryList1.get(i).getValue());
+        }
+
+        val segments = this.controller.getCurrentSegmentsForKeyValueTable(SCOPE,
+                newKVT.getKeyValueTableName()).join();
+
+        boolean deleted = this.controller.deleteKeyValueTable(SCOPE, newKVT.getKeyValueTableName()).join();
+        assertTrue("The requested KVT must get deleted", deleted);
+        log.info("KVT '{}' got deleted", newKVT.getKeyValueTableName());
+
+        assertFalse("Request to remove already deleted KVT must not succeed",
+                this.controller.deleteKeyValueTable(SCOPE, newKVT.getKeyValueTableName()).join());
+
+        for (val s : segments.getSegments()) {
+            AssertExtensions.assertSuppliedFutureThrows(
+                    "Segment " + s + " has not been deleted.",
+                    () -> this.tableStore.get(s.getKVTScopedName(), Collections.singletonList(new ByteArraySegment(new byte[1])), TIMEOUT),
+                    ex -> ex instanceof StreamSegmentNotExistsException);
+        }
+        log.info("Successfully completed test Delete KVT and check KVPs are not accessible");
+    }
+
+    /**
+     * Test creation of KVTable with zero (0) partition in their config value
+     */
+    @Test
+    @Ignore
```

Comment:
Why?

Reply:
@andreipaduroiu This test is maintained here to validate KeyValueTable creation with 0(zero) as partition size in order to validate an internal open issue once it gets fixed and merged into the master.

Reply:
Is there a GitHub Issue to track when this test will be reenabled? Otherwise there is a good chance this test will be forgotten forever.

Reply:
@andreipaduroiu There was an internal Jira I created to track this but now a new test addition request has already been raised, so I'll remove my test from here. Thanks
https://github.com/pravega/pravega/issues/5100